tools/wptrunner/wptrunner/formatters/chromium.py

```diff
@@ -121,10 +121,14 @@ def _store_test_result(self, name, actual, expected, actual_metadata,
         cur_dict = self.tests
         for name_part in name_parts:
             cur_dict = cur_dict.setdefault(name_part, {})
-        cur_dict["actual"] = actual
+        statuses = cur_dict.get("actual", "").split()
```

Comment:
introducing a split/join here seems suboptimal since it makes the runtime quadratic in the number of repetitions. That's probably not a huge problem in practice since the number of repetitions will typically be small, and I see that it avoids the need to post-process the `tests` dictionary when outputting at the end. But there should at least be a comment to explain the design choice.